packages/terra-notification-dialog/CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ Changelog
 
 Unreleased
 ----------
+### Fixed
+* Updated max-width to be `49%` for screen size greater than 768px.
+* Updated wdio snapshot
```

Comment:
This is not needed.

Reply:
You can remove the the `* Updated wdio snapshot`, it is not pertinent to the change from a consumers perspective.